app/src/main/res/values/strings.xml

```diff
@@ -69,6 +69,7 @@
     <string name="dialog_text_change_partner">Kletterpartner ändern</string>
     <string name="dialog_question_delete">Diesen Bereich löschen?</string>
     <string name="dialog_question_delete_ascend">Diese Begehung löschen?</string>
+    <string name="dialog_question_update">Dies aktualisiert alle Gebiete und kann daher einige Zeit dauern.</string>
```

Comment:
The text is a bit long. At least ob my screen the text is truncated. Either shorten the text or reduce the text size. However, the latter solution leads to an inconsistent layout.